amps/ags/rm-community/rm-community-repo/config/alfresco/module/org_alfresco_module_rm/alfresco-global.properties

```diff
@@ -139,3 +139,16 @@ content.metadata.async.extract.6.enabled=false
 
 # Max number of entries returned in Record search view
 rm.recordSearch.maxItems=500
+
+#
+# Hold bulk
+#
+rm.hold.bulk.threadCount=2
+rm.hold.bulk.maxItems=1000
+rm.hold.bulk.batchSize=100
+rm.hold.bulk.logging.interval.ms=1000
+rm.hold.bulk.itemsPerTransaction=1
```

Comment:
You wrote some great documentation for these in the PR description. Please include them here so that customers/partners (and we) have an easy place to see what each property does.

Reply:
Done.